otel/otel-config-multicluster.adoc

```diff
@@ -1,10 +1,8 @@
-// Module included in the following assemblies:
-// 
-// * otel/otel-configuring.adoc
-
 :_mod-docs-content-type: PROCEDURE
-[id="gathering-observability-data-from-different-clusters_{context}"]
-= Gathering the observability data from different clusters with the OpenTelemetry Collector
+[id="otel-gathering-observability-data-from-multiple-clusters"]
+= Gathering the observability data from multiple clusters
+include::_attributes/common-attributes.adoc[]
+:context: otel-gathering-observability-data-from-multiple-clusters
```

Comment:
Good - I would remove the product name from as many titles as possible tbh since the docs are already grouped per product in the left nav, and the abstract tends to refer to the product name anyway

Reply:
True. I was overcautious to differentiate it from the upstream product which has the same name. But yes, I'll revisit this point when preparing for the next release.